fury/ui.py

```diff
@@ -3146,24 +3152,32 @@ def __init__(self, labels, padding=1, font_size=18,
         position : (float, float)
             Absolute coordinates (x, y) of the lower-left corner of
             the button of the first option.
+        checked_labels: list(string)
+            List of labels that are checked on setting up.
         """
+
         self.labels = list(reversed(labels))
         self._padding = padding
         self._font_size = font_size
         self.font_family = font_family
+        self.checked_labels = []
+        self.checked_labels.extend(checked_labels)
         super(Checkbox, self).__init__(position)
         self.on_change = lambda checkbox: None
-        self.checked = []
 
     def _setup(self):
         """ Setup this UI component.
         """
         self.options = []
         button_y = self.position[1]
         for label in self.labels:
+            checked = bool(label in self.checked_labels)
+
             option = Option(label=label,
                             font_size=self.font_size,
-                            position=(self.position[0], button_y))
+                            position=(self.position[0], button_y),
+                            checked=checked)
+
```

Comment:
Instead of adding a `checked` parameter to `Option`, we could simply call `option.select()` here. Is there a good reason I fail to see that would require modifying the internal `Option` class?

Reply:
I think both solutions are OK.  In one case you need to add an `if` with `option.select()` and with the other case, no `if`, no function call, just an easy parameter.

Reply:
Fair enough.